.github/workflows/build.yml

```diff
@@ -81,23 +81,23 @@ jobs:
         Debug/luau-analyze tests/conformance/assert.lua
 
   coverage:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     steps:
     - uses: actions/checkout@v2
     - name: install
       run: |
         sudo apt install llvm
     - name: make coverage
       run: |
-        CXX=clang++ make -j2 config=coverage native=1 coverage
+        CXX=clang++-10 make -j2 config=coverage native=1 coverage
```

Comment:
I am assuming we are being explicit about the clang version here so we don't unintentionally run on an version of clang without the support we need?

Reply:
This is restoring the specific version we had earlier.
As far as I understand, later versions available on 20.04 generate worse coverage locations. It might be fixed in 14/15, but that needs 22.04.